src/database/setup.ts

```diff
@@ -24,16 +24,6 @@ const dbQuery = (query: string, successMessage: string, exitScript = false) => {
   });
 };
 
-//Creating a role for the user.
-const createRole = `CREATE ROLE ${process.env.DATABASE_USER} LOGIN CREATEDB`;
-dbQuery(createRole, `CREATED ROLE NAMED ${process.env.DATABASE_USER} SUCCESFULLY`);
-
-//Add password to role if provided
-if (process.env.DATABASE_PASSWORD) {
-  const addPassword = `ALTER ROLE ${process.env.DATABASE_USER} with PASSWORD ${process.env.DATABASE_PASSWORD}`;
-  dbQuery(addPassword, 'PASSWORD ADDED TO ROLE SUCCESSFULLY');
-}
-
```

Comment:
This seems like it should belong in a different PR. It has nothing to do with updating the user entity